core/dataset.h

```diff
@@ -157,6 +158,27 @@ struct ColumnIndex {
     /** Return the value of the column for all rows and timestamps. */
     virtual MatrixColumn getColumn(const ColumnName & column) const = 0;
 
+    /** Return a dense column, with one value for every row in the same order as
```

Comment:
No, the comment is right.  It means it should return them in the same order of rows as rowNames() (the method) returns.